packages/forest_admin_agent/spec/lib/forest_admin_agent/utils/schema/generator_action_spec.rb

```diff
@@ -397,6 +397,80 @@ module Schema
             end
           end
 
+          context 'with page element' do
+            before do
+              @collection = collection_build(
+                schema: {
+                  actions: {
+                    'Charge credit card' => BaseAction.new(
+                      scope: Types::ActionScope::SINGLE
+                    )
+                  }
+                },
+                get_form: [
+                  ActionLayoutElement::PageElement.new(
+                    next_button_label: proc { |_context| 'Next' },
```

Comment:
after speacking with Enky and Nico, we said theses fields are stirng only, no function allowed here right ?